packages/jest-util/src/__tests__/format_test_results_test.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+'use strict';
+
+const formatTestResults = require('../formatTestResults');
```

Comment:
Ah, because this was called `_test` but we only look for `.test` files, so this wasn't actually run. I'll fix it.